e2e_postgres_test.go

```diff
@@ -12,9 +12,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//go:build !skip_e2e && !skip_postgres
-// +build !skip_e2e,!skip_postgres
```

Comment:
Seems like we should leave `!skip_postgres`?

Reply:
Totally reasonable. Done.